test/foundry/unit/concrete/factory/TestK1ValidatorFactory_Deployments.tree

```diff
@@ -11,5 +11,7 @@ TestK1ValidatorFactory_Deployments
     │   └── it should return the expected address
     ├── when creating an account with the same owner and index
     │   └── it should result in the same address
-    └── when creating accounts with different indexes
-        └── it should result in different addresses
+    ├── when creating accounts with different indexes
```

Comment:
I think tests should confirm that you're able to compute same account address using the initData and salt index emitted in the events. 

or could be different params depending on the factory/method but you get the gist.

Reply:
directly assigning attesters = sorted; after sorting won’t work as expected because you cannot directly assign a memory array to a storage array this way in Solidity